opencti-platform/opencti-graphql/src/domain/connector.ts

```diff
@@ -316,6 +308,13 @@ export const deleteQueues = async (context, user) => {
 };
 // endregion
 
-export const queueDetails = async (connectorId) => {
-  return await getConnectorQueueDetails(connectorId);
+export const queueDetails = async (connectorId: string) => {
+  return getConnectorQueueDetails(connectorId);
+};
+
+export const connectorUser = async (context: AuthContext, user: AuthUser, userId: string) => {
+  if (isUserHasCapability(user, SETTINGS_SET_ACCESSES)) {
```

Comment:
this should be protected in the API side directly. See comment on opencti.graphql